docs/installation/index.md

```diff
@@ -5,10 +5,22 @@ sidebar: sidebars/documentation.html
 script: osmenu.js
 ---
 
+{% if site.version == null %}
+# Katello Nightly Installation
+{% else %}
 # Katello Installation
+{% endif %}
+
+{{ site.version }}
+
+{% if site.version == null %}
+  <div class="alert alert-danger">
+    These are the instructions for installing the unstable nightly release of katello!
```

Comment:
I think I like theforeman.org's way of doing things, if you're on anything but the latest stable version, it warns you and gives you a link to it - that would handle the nightly case, and everything else.

What do you think?

We have access to the info in site.latest: https://github.com/Katello/katello.org/blob/master/_config.yml#L5